include/onnxruntime/core/framework/op_node_proto_helper.h

```diff
@@ -10,20 +10,6 @@
 #include "core/common/gsl.h"
 #endif
 
-#ifdef __has_attribute
-#define ORT_HAVE_ATTRIBUTE(x) __has_attribute(x)
-#else
-#define ORT_HAVE_ATTRIBUTE(x) 0
-#endif
-
-#if ORT_HAVE_ATTRIBUTE(nodiscard)
-#define MUST_USE_RESULT [[nodiscard]]
```

Comment:
Hi Ed - just reading this and wanted to learn: was there a reason to remove this? Was it for readability?

Reply:
just some cleaning up - `[[nodiscard]]` is a standard attribute since C++17. also the `Status` class is already declared with `[[nodiscard]]` so we don't need to put it on a `Status` return type.